test/core_tests.js

```diff
@@ -78,6 +79,11 @@
       expected = "ease-in-out";
       assert.strictEqual( actual, expected, "canvas.style.transitionTimingFunction initialized correctly" );
 
+      // Ensure we specify perspective independent of transform for IE 11
+      actual = root.style.perspective;
+      expected = "";
+      assert.notStrictEqual( actual, expected, "root.style.perspective initialized correctly" );
```

Comment:
It looks like this test wouldn't fail only in IE but just serve as a verification that if we remove the `perspective` property it will fail in any browser, is that right?